src/style.css

```diff
@@ -1 +1,73 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.page {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't work. For example, `font-family: Avenir, Arial, sans-serif;`. This ensures text is still readable if the custom font fails to load.